packages/validate/test/ValidateMixin.test.js

```diff
@@ -19,7 +19,6 @@ const tagString = defineCE(
   class extends ValidateMixin(LionLitElement) {
```

Comment:
Any reason in particular we didn't use LitElement directly here?

Reply:
No idea, should probably change to LitElement ^^